ui_widgets.py

```diff
@@ -143,6 +143,27 @@ def validateOut(self, event):
             self.focus_set()
 
 
+class EntryHelp:
+    def __init__(self, master, text):
+        self.master = master
```

Comment:
When I click on a help button, the Main MAIA window places itself in front of the advanced config. This is unintuitive from the user perspective, as I have to then click back into the advanced config window to move it to front. Can this be fixed?
![image](https://github.com/Leading-AI-Development-Solutions/MAIA/assets/97774046/d014c270-42cf-4094-bae2-79115e20bbc5)

Reply:
rare mac w

Reply:
Fixed I think.

Reply:
Yes, resolved